.github/workflows/eden.yml

```diff
@@ -15,5 +15,5 @@ jobs:
     name: Execute Eden test workflow
     uses: ./.github/workflows/test.yml
     with:
-      eve_image: "lfedge/eve:11.7.0"
+      eve_image: "lfedge/eve:12.3.0"
```

Comment:
Bumping version here means we should bump versions in [default](https://github.com/lf-edge/eden/blob/e255da8d08c4f516d6da53fc7540299baa714594/pkg/defaults/defaults.go#L56), and perhaps we should update and merge https://github.com/lf-edge/eden/pull/969 before merging this, wdys @milan-zededa ?

Reply:
Yes, @christoph-zededa please update also default to 12.3.0.
Regarding #969, I'm debugging failing tests in https://github.com/lf-edge/eden/pull/981. Once I'm done I will update 969 or maybe close it and open new PR with all test fixes. So do not get this PR delayed because of 969.